cogment_verse/specs/spaces_serialization.py

```diff
@@ -42,13 +41,13 @@ def serialize_gym_space(gym_space, serilization_format=SerializationFormat.STRUC
             size = np.array(gym_space.n, dtype=np.dtype("int8"))
         return Space(multi_binary=MultiBinary(n=serialize_ndarray(size, serilization_format=serilization_format)))
 
-    if isinstance(gym_space, gym.spaces.MultiDiscrete):
+    if isinstance(gym_space, (gym.spaces.MultiDiscrete, gymnasium.spaces.MultiDiscrete)):
```

Comment:
For my task #134 , I'll make sure that we have a clearer separation of types of spaces for serialization and deserialization that will generalize to cases where alternate spaces are not identical to gym spaces. In this case no changes were really necessary to the deserialization since we do not care to use gymnasium observation/actions in the actor implementations. But in general we should deserialize to the same type that was initially serialized.

Reply:
@wduguay-air For instance, I keep both approach in this PR because as we discussed. We wanted to create another issue  to track the changes from gym to gymnasium.